src/Tests/FunctionalTests/CCW/Program.cs

```diff
@@ -527,4 +635,93 @@ public void Execute(object parameter)
     {
         throw new NotImplementedException();
     }
+}
+
+[BindableCustomProperty([nameof(Name), nameof(Value)], [typeof(int)])]
+partial class Language
+{
+    private readonly string[] _values = new string[4];
+
+    public string Name { get; } = "Language";
+    public int Value { get; set; }
+    public string this[int i]
+    {
+        get => _values[i];
+        set => _values[i] = value;
+    }
+}
+
+[global::WinRT.BindableCustomProperty([nameof(Name), nameof(Derived)], [typeof(int)])]

```

Comment:
just testing various attribute qualifiers here, and in line 660 and 666?

Reply:
Yea, including namespaces vs not.